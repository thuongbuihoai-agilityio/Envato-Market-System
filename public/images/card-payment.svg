<svg width="340" height="200" viewBox="0 0 340 200" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="340" height="200" rx="12" fill="#1A202C"/>
<mask id="mask0_1557_18598" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="317" height="190">
<rect width="317" height="190" rx="16" fill="#194BFB"/>
</mask>
<g mask="url(#mask0_1557_18598)">
<path d="M144 95.5C144 177.514 210.486 244 292.5 244C374.514 244 441 177.514 441 95.5C441 13.4857 374.514 -53 292.5 -53C210.486 -53 144 13.4857 144 95.5ZM399.611 95.5C399.611 154.656 351.656 202.611 292.5 202.611C233.344 202.611 185.389 154.656 185.389 95.5C185.389 36.344 233.344 -11.6115 292.5 -11.6115C351.656 -11.6115 399.611 36.344 399.611 95.5Z" fill="url(#paint0_linear_1557_18598)" fill-opacity="0.9"/>
<path d="M238 95C238 128.137 264.863 155 298 155C331.137 155 358 128.137 358 95C358 61.8629 331.137 35 298 35C264.863 35 238 61.8629 238 95ZM333.209 95C333.209 114.446 317.446 130.209 298 130.209C278.554 130.209 262.791 114.446 262.791 95C262.791 75.5544 278.554 59.7906 298 59.7906C317.446 59.7906 333.209 75.5544 333.209 95Z" fill="url(#paint1_linear_1557_18598)" fill-opacity="0.9"/>
</g>
<rect x="35.25" y="31.1079" width="31.5" height="23.8629" rx="3.75" fill="url(#paint2_linear_1557_18598)" stroke="#9F7E3B" stroke-width="0.5"/>
<path d="M35.5 38.3524H44C45.1046 38.3524 46 39.2478 46 40.3524V44.2231M46 54.7403V50.0937M46 50.0937V44.2231M46 50.0937H35.5M46 44.2231H35.5" stroke="black" stroke-opacity="0.76" stroke-width="0.5"/>
<path d="M66.5 38.3524H56.5M56.5 38.3524V44.223M56.5 38.3524V31.3575M56.5 54.7402V50.0937M56.5 50.0937V44.223M56.5 50.0937H66.5M56.5 44.223H66.5" stroke="black" stroke-opacity="0.76" stroke-width="0.5"/>
<path opacity="0.6" d="M236 37.1018C236 34.2053 238.38 31.8572 241.316 31.8572C244.252 31.8572 246.631 34.2053 246.631 37.1018V38.521C245.741 38.1695 244.77 37.9759 243.752 37.9759C239.471 37.9759 236 41.4002 236 45.6244V37.1018Z" fill="white"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M243.732 53.3501C248.002 53.3501 251.464 49.8909 251.464 45.6238C251.464 41.3567 248.002 37.8976 243.732 37.8976C239.462 37.8976 236 41.3567 236 45.6238C236 49.8909 239.462 53.3501 243.732 53.3501ZM243.732 49.5973C245.928 49.5973 247.708 47.8183 247.708 45.6238C247.708 43.4293 245.928 41.6503 243.732 41.6503C241.536 41.6503 239.756 43.4293 239.756 45.6238C239.756 47.8183 241.536 49.5973 243.732 49.5973Z" fill="white"/>
<path d="M255.715 48.2755V37.4588H259.743C260.326 37.4588 260.854 37.5798 261.328 37.8219C261.802 38.0589 262.176 38.3988 262.449 38.8418C262.728 39.2848 262.867 39.805 262.867 40.4025C262.867 40.6446 262.833 40.8944 262.766 41.1519C262.705 41.4095 262.596 41.6464 262.442 41.8627C262.292 42.0791 262.086 42.2439 261.823 42.3572C262.163 42.4396 262.437 42.5942 262.643 42.8208C262.849 43.0423 263.006 43.2998 263.114 43.5934C263.223 43.8819 263.295 44.178 263.331 44.4819C263.367 44.7858 263.385 45.0614 263.385 45.3087C263.385 45.8649 263.241 46.3672 262.952 46.8153C262.663 47.2634 262.274 47.6188 261.784 47.8815C261.3 48.1442 260.764 48.2755 260.176 48.2755H255.715ZM257.849 46.4599H259.952C260.235 46.4599 260.493 46.3929 260.725 46.259C260.957 46.1199 261.143 45.9319 261.282 45.695C261.421 45.4529 261.491 45.1799 261.491 44.876C261.491 44.6184 261.421 44.3738 261.282 44.142C261.143 43.9102 260.957 43.7222 260.725 43.578C260.493 43.4286 260.235 43.3539 259.952 43.3539H257.849V46.4599ZM257.849 41.5769H259.496C259.836 41.5769 260.127 41.4661 260.369 41.2446C260.612 41.0232 260.733 40.7373 260.733 40.387C260.733 40.0265 260.612 39.7535 260.369 39.5681C260.127 39.3775 259.836 39.2822 259.496 39.2822H257.849V41.5769Z" fill="white"/>
<path d="M269.7 40.5493H271.834V48.2755H269.692L269.592 47.472C269.391 47.7708 269.133 48.0128 268.819 48.1983C268.504 48.3837 268.136 48.4764 267.713 48.4764C267.136 48.4764 266.597 48.3683 266.097 48.1519C265.597 47.9356 265.156 47.6368 264.775 47.2557C264.393 46.8745 264.094 46.4341 263.878 45.9345C263.667 45.4297 263.561 44.8889 263.561 44.312C263.561 43.7608 263.662 43.2457 263.863 42.7667C264.064 42.2877 264.347 41.8653 264.713 41.4996C265.079 41.1339 265.499 40.848 265.973 40.642C266.453 40.436 266.968 40.333 267.52 40.333C267.994 40.333 268.419 40.4334 268.796 40.6343C269.177 40.8352 269.509 41.0901 269.793 41.3992L269.7 40.5493ZM267.659 46.429C268.009 46.429 268.326 46.3388 268.61 46.1585C268.893 45.9783 269.118 45.7362 269.283 45.4323C269.448 45.1232 269.53 44.7833 269.53 44.4124C269.53 44.0364 269.448 43.6964 269.283 43.3925C269.118 43.0886 268.893 42.8466 268.61 42.6663C268.326 42.486 268.009 42.3959 267.659 42.3959C267.303 42.3959 266.981 42.486 266.692 42.6663C266.404 42.8466 266.174 43.0912 266.004 43.4003C265.834 43.7042 265.749 44.0416 265.749 44.4124C265.749 44.7833 265.834 45.1232 266.004 45.4323C266.18 45.7362 266.412 45.9783 266.7 46.1585C266.989 46.3388 267.308 46.429 267.659 46.429Z" fill="white"/>
<path d="M280.107 43.3616V48.2755H277.973V43.9488C277.973 43.6758 277.906 43.4312 277.772 43.2148C277.643 42.9934 277.468 42.8182 277.246 42.6895C277.03 42.5607 276.787 42.4963 276.519 42.4963C276.251 42.4963 276.006 42.5607 275.785 42.6895C275.563 42.8182 275.388 42.9934 275.259 43.2148C275.13 43.4312 275.066 43.6758 275.066 43.9488V48.2755H272.932L272.924 40.5493H275.058L275.066 41.121C275.323 40.8738 275.625 40.6806 275.97 40.5416C276.321 40.4025 276.689 40.333 277.076 40.333C277.638 40.333 278.146 40.4694 278.599 40.7424C279.058 41.0154 279.424 41.3811 279.697 41.8396C279.97 42.2928 280.107 42.8002 280.107 43.3616Z" fill="white"/>
<path d="M288.379 48.2755H285.92L284.281 45.4555L283.338 46.5758V48.2755H281.204V36.6862H283.338V43.2612L285.611 40.5493H288.395L285.742 43.7248L288.379 48.2755Z" fill="white"/>
<path d="M294.237 48.4762C293.505 48.4762 292.817 48.332 292.173 48.0435C291.528 47.7499 290.961 47.3456 290.472 46.8305C289.987 46.3154 289.606 45.7205 289.327 45.0458C289.054 44.3658 288.918 43.6396 288.918 42.867C288.918 42.0892 289.054 41.3629 289.327 40.6881C289.606 40.0082 289.987 39.4133 290.472 38.9034C290.961 38.3883 291.528 37.9865 292.173 37.6981C292.817 37.4045 293.505 37.2577 294.237 37.2577C295.114 37.2577 295.92 37.4611 296.657 37.8681C297.394 38.2698 298.005 38.8081 298.49 39.4829L296.588 40.5104C296.299 40.1705 295.951 39.9001 295.544 39.6992C295.137 39.4932 294.701 39.3901 294.237 39.3901C293.799 39.3901 293.387 39.4803 293 39.6606C292.614 39.8408 292.273 40.0907 291.98 40.41C291.691 40.7294 291.464 41.0976 291.299 41.5149C291.134 41.9321 291.052 42.3828 291.052 42.867C291.052 43.346 291.134 43.7967 291.299 44.219C291.464 44.6363 291.691 45.0045 291.98 45.3239C292.273 45.6381 292.614 45.8879 293 46.0733C293.387 46.2536 293.799 46.3438 294.237 46.3438C294.706 46.3438 295.142 46.2407 295.544 46.0347C295.951 45.8287 296.299 45.5557 296.588 45.2157L298.49 46.251C298.005 46.9207 297.394 47.4589 296.657 47.8658C295.92 48.2728 295.114 48.4762 294.237 48.4762Z" fill="white"/>
<path d="M302.449 48.4762C301.738 48.4762 301.088 48.2934 300.501 47.9276C299.918 47.5619 299.452 47.0726 299.101 46.4597C298.756 45.8416 298.583 45.1591 298.583 44.4122C298.583 43.8456 298.684 43.3176 298.885 42.8283C299.086 42.3338 299.362 41.9012 299.712 41.5303C300.068 41.1543 300.48 40.8607 300.949 40.6495C301.418 40.4383 301.918 40.3327 302.449 40.3327C303.161 40.3327 303.807 40.5156 304.39 40.8813C304.978 41.247 305.444 41.7389 305.789 42.357C306.14 42.9751 306.315 43.6602 306.315 44.4122C306.315 44.9736 306.215 45.499 306.014 45.9884C305.813 46.4777 305.534 46.9104 305.179 47.2864C304.828 47.6572 304.418 47.9483 303.949 48.1594C303.485 48.3706 302.985 48.4762 302.449 48.4762ZM302.449 46.3438C302.779 46.3438 303.076 46.2562 303.338 46.0811C303.601 45.9008 303.807 45.6639 303.957 45.3703C304.106 45.0767 304.181 44.7573 304.181 44.4122C304.181 44.0568 304.101 43.7323 303.941 43.4387C303.787 43.1399 303.578 42.903 303.315 42.7279C303.052 42.5528 302.764 42.4652 302.449 42.4652C302.124 42.4652 301.831 42.5553 301.568 42.7356C301.305 42.9159 301.096 43.1528 300.941 43.4464C300.792 43.74 300.717 44.0619 300.717 44.4122C300.717 44.7779 300.795 45.1076 300.949 45.4012C301.109 45.6896 301.32 45.9188 301.583 46.0888C301.846 46.2588 302.135 46.3438 302.449 46.3438Z" fill="white"/>
<path d="M307.851 48.4762C307.552 48.4762 307.297 48.3732 307.085 48.1672C306.879 47.956 306.776 47.6984 306.776 47.3945C306.776 47.1009 306.879 46.8485 307.085 46.6374C307.297 46.4262 307.552 46.3206 307.851 46.3206C308.15 46.3206 308.405 46.4262 308.616 46.6374C308.828 46.8485 308.933 47.1009 308.933 47.3945C308.933 47.6984 308.828 47.956 308.616 48.1672C308.405 48.3732 308.15 48.4762 307.851 48.4762Z" fill="white"/>
<path d="M290.528 145.428C289.296 144.987 287.994 144.765 286.683 144.774C282.444 144.774 279.45 146.876 279.429 149.894C279.402 152.111 281.556 153.36 283.186 154.101C284.858 154.862 285.417 155.342 285.411 156.023C285.401 157.058 284.076 157.528 282.844 157.528C281.151 157.528 280.226 157.295 278.803 156.714L278.276 156.471L277.675 159.945C278.705 160.368 280.563 160.73 282.481 160.762C286.988 160.762 289.934 158.673 289.969 155.462C290.006 153.691 288.84 152.353 286.387 151.247C284.895 150.53 283.97 150.05 283.97 149.322C283.97 148.678 284.762 147.99 286.418 147.99C287.52 147.963 288.616 148.167 289.632 148.587L290.03 148.762L290.631 145.405L290.528 145.428ZM301.529 145.055H298.217C297.186 145.055 296.41 145.335 295.96 146.341L289.59 160.545H294.097L294.998 158.216L300.498 158.221C300.628 158.766 301.026 160.545 301.026 160.545H305L301.529 145.055ZM273.332 144.925H277.627L274.94 160.423H270.651L273.332 144.92V144.925ZM262.429 153.467L262.874 155.618L267.071 145.055H271.62L264.861 160.524H260.322L256.613 147.426C256.553 147.21 256.412 147.025 256.218 146.907C254.881 146.227 253.465 145.712 252 145.371L252.058 145.045H258.971C259.909 145.082 260.664 145.371 260.916 146.357L262.426 153.475V153.467H262.429ZM296.23 155.047L297.941 150.713C297.92 150.759 298.294 149.818 298.511 149.236L298.805 150.574L299.799 155.045H296.227V155.047H296.23Z" fill="white"/>
<g opacity="0.6">
</g>
<defs>
<linearGradient id="paint0_linear_1557_18598" x1="95.5" y1="95" x2="231.5" y2="95" gradientUnits="userSpaceOnUse">
<stop stop-color="white" stop-opacity="0.15"/>
<stop offset="1" stop-color="white" stop-opacity="0"/>
</linearGradient>
<linearGradient id="paint1_linear_1557_18598" x1="216.5" y1="95" x2="323" y2="95" gradientUnits="userSpaceOnUse">
<stop stop-color="white" stop-opacity="0.15"/>
<stop offset="1" stop-color="white" stop-opacity="0"/>
</linearGradient>
<linearGradient id="paint2_linear_1557_18598" x1="35" y1="30.8579" x2="66.9825" y2="55.3626" gradientUnits="userSpaceOnUse">
<stop stop-color="#E6AC3C"/>
<stop offset="1" stop-color="#FFDA93"/>
</linearGradient>
</defs>
</svg>
