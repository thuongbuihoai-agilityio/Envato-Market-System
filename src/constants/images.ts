export const IMAGES = {
  SIGN_IN: {
    url: '/icons/signin.svg',
    alt: 'This is sign in image',
  },
  SIGN_UP: {
    url: '/icons/signup.svg',
    alt: 'This is sign in image',
  },
  SQUARE: {
    url: '/icons/square.svg',
    alt: 'This is square image',
  },
  VLINE: {
    url: '/icons/vline.svg',
    alt: 'This is vline image',
  },
  DOTTED: {
    url: '/icons/dotted.svg',
    alt: 'This is dotted image',
  },
  GOOGLE: {
    url: '/icons/google.svg',
    alt: 'This is the logo Google',
  },
  APPLE: {
    url: '/icons/apple.svg',
    alt: 'This is the logo Apple',
  },

  DEBIT_ICON: {
    url: '/icons/debit-icon.svg',
    alt: 'This is the logo debit card',
  },
  AVATAR: {
    url: '/images/avatar.webp',
    alt: 'This is avatar',
  },
  USER_AVATAR: {
    url: '/icons/user-avatar.webp',
    alt: 'This is the user avatar',
  },
  TOTAL_EARN: {
    url: '/icons/total-earn.svg',
    alt: 'Total earn icon',
  },
  GROWTH: {
    url: '/icons/growth.svg',
    alt: 'Growth icon',
  },
  LEFT_ARROW: {
    url: '/icons/left-arrow.svg',
    alt: 'This is the left arrow image',
  },
  LOGO_MINI_LIGHT: {
    url: '/icons/logo-mini-light.svg',
    alt: 'This is the logo mini image',
  },
  LOGO_MINI_DARK: {
    url: '/icons/logo-mini-dark.svg',
    alt: 'This is the logo mini image',
  },
  DASHBOARD: {
    url: '/icons/dashboard.svg',
    alt: 'This is the dashboard image',
  },
  TRANSACTION: {
    url: '/icons/transaction.svg',
    alt: 'This is the transaction image',
  },
  STATISTIC: {
    url: '/icons/statistics.svg',
    alt: 'This is the statistics image',
  },
  ANALYTICS: {
    url: '/icons/analytics.svg',
    alt: 'This is the analytics image',
  },
  MY_WALLET: {
    url: '/icons/my-wallet.svg',
    alt: 'This is the my wallet image',
  },
  INBOX: {
    url: '/icons/inbox.svg',
    alt: 'This is the inbox image',
  },
  NOTE: {
    url: '/icons/note.svg',
    alt: 'This is the note image',
  },
  INTEGRATIONS: {
    url: '/icons/integrations.svg',
    alt: 'This is the integration image',
  },
  USER: {
    url: '/icons/user.svg',
    alt: 'This is the user image',
  },
  CALENDAR: {
    url: '/icons/calendar.svg',
    alt: 'This is the calendar image',
  },
  HISTORY: {
    url: '/icons/history.svg',
    alt: 'This is the history image',
  },
  SUPPORT: {
    url: '/icons/support.svg',
    alt: 'This is the support image',
  },
  SETTING: {
    url: '/icons/setting.svg',
    alt: 'This is the setting image',
  },
  COMING_SOON: {
    url: '/icons/coming-soon.svg',
    alt: 'This is the coming soon image',
  },
  NOT_FOUND: {
    url: '/images/not-found.svg',
    alt: 'This is the not found image',
  },
  SIGN_OUT: {
    url: '/icons/sign-out.svg',
    alt: 'This is the sign out image',
  },
  CASHBACK: {
    url: '/images/cashback.webp ',
    alt: 'This is the cashback image',
  },
  MONEY_BAG: {
    url: '/icons/money-bag.svg',
    alt: 'This is the money bag image',
  },
  RIGHT_ARROW: {
    url: '/icons/right-arrow.svg',
    alt: 'This is the right arrow image',
  },
  INBOX_AVATAR: {
    url: '/images/avatar.webp',
    alt: 'This is the inbox avatar image',
  },
  MICRO: {
    url: '/icons/micro.svg',
    alt: 'Micro icon',
  },

  ATTACH: {
    url: '/icons/attach.svg',
    alt: 'Attach icon',
  },

  SEND: {
    url: '/icons/send.svg',
    alt: 'Send icon',
  },

  CHAT_USER_AVATAR: {
    url: '/images/avatar-user-chat.webp',
    alt: 'This is the user avatar',
  },

  INCREASE_ICON: {
    url: '/icons/increase.svg',
    alt: 'This is the increase icon',
  },

  CARD_PAYMENT: {
    url: '/images/card-payment.svg',
    alt: 'This is the card payment icon',
  },

  BIG_AVATAR: {
    url: '/images/big-avatar.webp',
    alt: 'This is the big avatar',
  },

  EDIT: {
    url: '/icons/edit.svg',
    alt: 'Edit icon',
  },

  AVATAR_SIGN_UP: {
    url: '/images/avatar-sign-up.webp',
    alt: 'This is avatar sign up',
  },
};
