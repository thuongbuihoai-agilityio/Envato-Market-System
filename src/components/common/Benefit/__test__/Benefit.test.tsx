import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

// component
import { Benefit } from '@components/index';
import { IMAGES } from '@constants/images';

test('renders Benefit with content', () => {
  const { container } = render(
    <Benefit imageURL={IMAGES.SIGN_IN.url} alt={IMAGES.SIGN_IN.alt} />,
  );
  expect(container).toMatchSnapshot();
});
