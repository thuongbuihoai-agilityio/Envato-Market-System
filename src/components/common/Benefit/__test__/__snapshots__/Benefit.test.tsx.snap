// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Benefit with content 1`] = `
<div>
  <div
    class="css-1nyw6b"
  >
    <img
      alt="This is sign in image"
      class="chakra-image css-0"
      src="/icons/signin.svg"
    />
    <img
      alt="This is square image"
      class="chakra-image css-nuaex1"
      src="/icons/square.svg"
    />
    <img
      alt="This is vline image"
      class="chakra-image css-bvz8js"
      src="/icons/vline.svg"
    />
    <img
      alt="This is dotted image"
      class="chakra-image css-12wsc5z"
      src="/icons/dotted.svg"
    />
    <div
      class="css-1wwt0r"
    >
      <h3
        class="chakra-heading css-tmruzz"
      >
        Speady, Easy and Fast
      </h3>
      <p
        class="chakra-text css-1rphqkz"
      >
        BankCo. help you set saving goals, earn cash back offers, Go to disclaimer for more details and get paychecks up to two days early. Get a
        <mark
          class="css-1rn2hzh"
        >
          $20
        </mark>
        bonus when you receive qualifying direct deposits
      </p>
    </div>
  </div>
</div>
`;
